data/interfaces/default/graphs.html

```diff
@@ -258,9 +335,10 @@
             var m = dateValue.getMonth() + 1;
             var y = dateValue.getFullYear();
             var dateString = '' + y + '-' + (m<=9 ? '0' + m : m) + '-' + (d <= 9 ? '0' + d : d);
-
+            var u = '';
+            //var u = $.cookie("example") == 'tabs4') ? 'get_addedat_modal': 'history_table_modal'
             $.ajax({
-                "url": "history_table_modal",
+                "url": "get_addedat_modal",
```

Comment:
Ok, but you click the "correct" date? So clicking on 20.10.2015 and it passes 19.10.2015 to the selecthandle??